ra/ra.go

```diff
@@ -377,12 +382,25 @@ func NewValidationProfiles(defaultName string, configs map[string]*ValidationPro
 			}
 		}
 
+		var identifierTypes []identifier.IdentifierType
+		for _, identType := range config.IdentifierTypes {
+			identifierTypes = append(identifierTypes, identifier.IdentifierType(identType))
+		}
+		// If this profile has no identifier types configured, default to DNS.
+		// This default is temporary, to improve deployability.
+		//
+		// TODO(#8184): Remove this default.
+		if len(identifierTypes) == 0 {
+			identifierTypes = append(identifierTypes, identifier.TypeDNS)
```

Comment:
Less opportunity for future bugs:
```suggestion
			identifierTypes = []identifier.IdentifierType{identifier.TypeDNS}
```